MultiplayerModFiles/Launcher/RunGame.py

```diff
@@ -584,16 +584,21 @@ def Init():
     portal2path = FindInConfig(configdata, "portal2path")
     
     # if we don't have a path, find it
-    if (portal2path == "undefined"):
+    if (portal2path == "undefined") or ((os.path.exists(portal2path)) != True) or (os.path.exists(portal2path + nf + "portal2_dlc2") != True):
```

Comment:
check if the path of the game in the config file is not given, doesn't exist or the game is not there
if any of the checks is false it will ask the user to give the path of the game
this way we can make sure that even if the user altered the config file, *i did*, the mod will work fine